screens/PreviewDirections.js

```diff
@@ -168,26 +173,28 @@ function PreviewDirections (props) {
     const destination = `${toCoordinates.latitude},${toCoordinates.longitude}`;
     // The variables retrived from the preference page 
 
+ 
     /**
-         * Description: fetchData() is an async method that makes the API request to Google Maps.
-         * Particularity: Requires origin, destination latitudes and longitudes as well the API key. 
-         * @param {*} transportType 
-        */
+     * Description: fetchData() is an async method that makes the API request to Google Maps.
+     * Particularity: Requires origin, destination latitudes and longitudes as well the API key. 
+     * @param {*} transportType 
+    */
     const fetchData = async () => {
         try {
             // Retrieving the apiKey from the AsyncStorage.
             let keyId = await AsyncStorage.getItem("apiKeyId");
-            let resp = await fetch(`https://maps.googleapis.com/maps/api/directions/json?origin=${origin}&destination=${destination}&key=${keyId}&mode=${outdoorTransportType}`);
+            let resp = await fetch(`https://maps.googleapis.com/maps/api/directions/json?origin=${origin}&destination=${destination}&key=${keyId}&mode=${transportType}`);
             const jsonResponse = await resp.json();
             if (jsonResponse && jsonResponse.routes.length >= 1) { //Added for better error handling. A.U
                 const decodedPoints = decodedPolylinesAlgo(jsonResponse.routes[0].overview_polyline.points);
                 setDecodedPolylines(decodedPoints);
                 updateMapRegionToOverallPath(decodedPoints);
                 //Command Pattern
-                let filteredInstruction = getFilteredDetailedInstructions(jsonResponse.routes[0].legs[0], outdoorTransportType);
+                let filteredInstruction = getFilteredDetailedInstructions(jsonResponse.routes[0].legs[0], transportType);
                 filteredInstruction.generalRouteInfo.overviewPolyline = decodedPoints;
                 filteredInstruction.generalRouteInfo.isStartAddressClassRoom = fromCoordinates.isClassRoom ? fromCoordinates.isClassRoom : null;
                 filteredInstruction.generalRouteInfo.isEndAddressClassRoom = toCoordinates.isClassRoom ? toCoordinates.isClassRoom : null;
+                // console.log(filteredInstruction.steps);
```

Comment:
Do you need this ?

Reply:
> the onPress on the map to make the drop down list disappear is not a responsive 😥
> 
> It works a times or sometime I feel like I have to press on specific spots on the map

It's legit the best we can do for the onPress event... Any other solution would have to use another event.